lib/reek/report/report.rb

```diff
@@ -129,9 +130,9 @@ def show
     class HTMLReport < Base
       require 'erb'
 
-      def show(target_path = 'reek.html')
-        template_path = File.expand_path('../html_report.html.erb', __FILE__)
-        File.write target_path, ERB.new(File.read(template_path)).result(binding)
+      def show(target_path = Pathname('reek.html'))
+        template_path = Pathname("#{__dir__}/html_report.html.erb")
```

Comment:
We have three options:

1. `Pathname.new(__dir__).join('html_report.html.erb')` – your preference
1. `Pathname.new("#{__dir__}/html_report.html.erb")` – @troessner’s preference
1. `Pathname.new(__dir__) + 'html_report.html.erb'` – the one I used initially (but I’m ok with either)

Ruby 2.2 adds `Pathname.new(__dir__) / 'html_report.html.erb'` but we can’t use it yet. :)

If you two could figure out which one you want I’ll change all places accordingly. :)